src/main/java/tools/map/xml/creator/CanalDefinitionsPanel.java

```diff
@@ -81,14 +81,14 @@ public static void layout(final MapXmlCreator mapXmlCreator) {
 
   @Override
   protected void paintCenterSpecifics(final Graphics g, final String centerName, final FontMetrics fontMetrics,
-      final Point item, final int xTextStart) {
+      final Point item, final int textStartX) {
```

Comment:
Meh, not really happy with this naming convention, but it is used in many other places throughout the code (i.e. `<something>X` or `<something>Y`).  Any better suggestions are welcome.